Xamarin.Forms.Platform.UAP/CollectionView/ItemsViewRenderer.cs

```diff
@@ -354,6 +369,44 @@ async Task AnimateTo(ListViewBase list, object targetItem, ScrollToPosition scro
 			//}
 		}
 
+		void UpdateVerticalScrollBarVisibility()
+		{
+			if (_defaultVerticalScrollVisibility == null)
+				_defaultVerticalScrollVisibility = ScrollViewer.GetVerticalScrollBarVisibility(Control);
+
+			switch (Element.VerticalScrollBarVisibility)
+			{
+				case (ScrollBarVisibility.Always):
+					ScrollViewer.SetVerticalScrollBarVisibility(Control, UwpScrollBarVisibility.Visible);
+					break;
+				case (ScrollBarVisibility.Never):
+					ScrollViewer.SetVerticalScrollBarVisibility(Control, UwpScrollBarVisibility.Hidden);
+					break;
+				case (ScrollBarVisibility.Default):
+					ScrollViewer.SetVerticalScrollBarVisibility(Control, (UwpScrollBarVisibility)_defaultVerticalScrollVisibility);
```

Comment:
I think instead of casting this to `UwpScrollBarVisibility`, we should just be using `_defaultVerticalScrollVisibility.Value` here.